src/Security/OpenIdConfigurationProvider.php

```diff
@@ -192,10 +197,32 @@ public function getAuthorizationUrl(array $options = []): string
      *
      * @throws ItkOpenIdConnectException
      */
-    public function validateIdToken(string $idToken, string $nonce): object
+
```

Comment:
You have duplicated the docblock for `validateIdToken`, please merge the two blocks

Reply:
Fixed in 860c85a